src/utils/bytes.cairo

```diff
@@ -32,8 +33,77 @@ func felt_to_ascii{range_check_ptr}(n: felt) -> (ascii_len: felt, ascii: felt*)
     let ascii_len = [ap - 1];
     let ascii = cast([fp], felt*);
 
-    let (rev: felt*) = alloc();
-    reverse(rev, ascii_len, ascii);
+    reverse(dst, ascii_len, ascii);
 
-    return (ascii_len, rev);
+    return ascii_len;
+}
+
+func bytes_to_bytes8_little_endian(dst: felt*, bytes_len: felt, bytes: felt*) -> felt {
```

Comment:
why are we returning a felt, and then not using it?

Reply:
looks like it's not used all the time. This function is just used with keccak things requiring the intput to be encoded as such, but using the initial bytes_len